geode-lucene/src/main/java/org/apache/geode/cache/lucene/LuceneResultStruct.java

```diff
@@ -39,6 +41,8 @@
    * Return score the score of the entry matching the query. Scores are computed by Lucene based on
    * how closely documents match the query.
    *
+   * @return float value representing the score of the document / entry obtained as a result of
```

Comment:
what does document / entry mean in this case? We should probably just use entry instead of document here?